File: app/src/main/java/com/xrest/hamrobazar/ResponseClass/Message.kt
package com.xrest.hamrobazar.ResponseClass

data class Message(
        val _id:String?=null,
    val Sender:Users?=null,
    val Reciever:Users?=null,
val Messages:MutableList<InnerMessage>?=null


)
data class InnerMessage(
        val val_id:String?=null,
        val message:String?=null,
    val user:Users?=null,
        val format:String?=null
)

File: newBackend/models/message.js
const mongoose = require('mongoose')


const chat = mongoose.model('Chat',{


Sender:{
    type:mongoose.Schema.Types.ObjectId,
    ref:'User'
},
Reciever:{
    type:mongoose.Schema.Types.ObjectId,
    ref:'User'
},
Messages:[
    {
message:{
    type:String
},
user:{
    type:mongoose.Schema.Types.ObjectId,
    ref:'User'
},
format:{
    type:String
},

    }
]




})
module.exports = chat